docs/admin/50-resources/10-whats-new.md

```diff
@@ -4,6 +4,20 @@ id: whats-new
 title: "What's New"
 ---
 
+# 2025 March 26
+
+**Release notes OpenCloud 2.0.0 (Production release)**
+
+- **🌲 File Native Backups with PosixFS**: OpenCloud introduces PosixFS to allow for easy file native backups. As all files and metadata information is stored directly on the storage (no database needed), you can create standard file-level snapshots or copies. <br />
+Previously, files and folders were stored in a hash-based structure, making the layout unreadable for humans.With PosixFS, the full file tree is directly visible on the storage, providing transparency and confidence that all data is being backed up correctly. With this, we would like that admins benefit from easier maintenance, fewer moving parts, and improved fault tolerance in backup and disaster recovery scenarios. <br />
+**Note:** After adding, writing, or restoring files directly into the OpenCloud storage, a restart of OpenCloud is required for changes to be recognized. A future feature, currently in development, called “collaborative storage” will allow external file changes (e.g., via CLI) to be detected in real time by OpenCloud and its clients.
```

Comment:
The docs say that the admin should stop OpenCloud first, then adjust files in the storage, then start OpenCloud.

This note implies that you can have OpenCloud running, adjust files in the storage while OpenCloud is running, and then restart OpenCloud to recognize the storage changes - is that a safe way to do it?

Reply:
good point, thanks!